libr/core/cmd_info.c

```diff
@@ -211,38 +211,45 @@ static void r_core_file_info(RCore *core, int mode) {
 		if (dbg) {
 			dbg = R_PERM_WX;
 		}
-		if (desc) {
-			pair ("blksz", sdb_fmt ("0x%"PFMT64x, (ut64) core->io->desc->obsz));
-		}
-		pair ("block", sdb_fmt ("0x%x", core->blocksize));
 		if (desc) {
 			pair ("fd", sdb_fmt ("%d", desc->fd));
 		}
 		if (fn || (desc && desc->uri)) {
 			pair ("file", fn? fn: desc->uri);
 		}
+		if (desc) {
+			ut64 fsz = r_io_desc_size (desc);
+			if (fsz != UT64_MAX) {
+				char humansz[8];
+				pair ("size", sdb_itoca (fsz));
+				r_num_units (humansz, sizeof (humansz), fsz);
+				pair ("humansz", humansz);
+			}
+		}
+		if (desc) {
+			pair ("mode", r_str_rwx_i (desc->perm & R_PERM_RWX));
+		}
 		if (plugin) {
 			pair ("format", plugin->name);
 		}
 		if (desc) {
 			pair ("iorw", r_str_bool (io_cache || desc->perm & R_PERM_W));
-			pair ("mode", r_str_rwx_i (desc->perm & R_PERM_RWX));
 		}
+		if (desc) {
+			pair ("blksz", sdb_fmt ("0x%"PFMT64x, (ut64) core->io->desc->obsz));
+		}
+		pair ("block", sdb_fmt ("0x%x", core->blocksize));
+		
```

Comment:
so many empty lines

Reply:
The empty lines are removed in Update cmd_info.c commit